### PR TITLE
API Complete

### PR DESCRIPTION
**Pull request by DarianAltF4**
Additions:
- All endpoints added and working correctly. So far... writing tests next.
- Fixed shema issue. I was missing the reviewers email.
- Fixed ETL issue. If the response was null it would save null as a string. Now saves as an empty string like the legacy API.

### CODE REVIEW

**Comment by DarianAltF4** [comment]
Not really 2000+ lines of code. Most of it is a package.json lock.

File: NoSQL-DB/no-sql-db.js [hunk 1]
```diff
@@ -71,4 +71,33 @@ const getAllReviews = (product_id) => {
...
\ No newline at end of file
+const postReview = (params) => {
+  console.log(params);
+  let product_id = params.product_id;
+  let rating = params.rating;
+  let summary = params.summary;
+  let body = params.body;
+  let recommended = params.recommended;
+  let name = params.name;
+  let email = params.email;
```
Comment (ahmadalmomani96) [inline NoSQL-DB/no-sql-db.js:84]: This can be done in fewer lines of code with array/object desctructuring.

**Comment by ahmadalmomani96** [comment]
Hi Darian, 

I went through your code and it's pretty solid. I left one comment which i believe get eliminated through further commits you made. I'll also note that at first the commits you made where very small and very focused which I believe is preferable compared to larger more unstructured commits. 

Keep up the good work. 

Ahmad Almomani

File: NoSQL-DB/no-sql-db.js (hunk 1 above)
Comment (DarianAltF4) [reply NoSQL-DB/no-sql-db.js:84]: Yeah. I agree. For some reason when I try to do the destructing, my post gives me an error saying that my function is undefined. So I ended up doing it the old school way... I just tried it again just to double check and it looks like it is causing the same error. I am not sure why. I did the very same thing in my etl.js 😅

**Pull request merged**